src/react/components/pages/predict/predictPage.tsx

```diff
@@ -339,10 +430,48 @@ export default class PredictPage extends React.Component<IPredictPageProps, IPre
                     when={this.state.isPredicting}
                     message={"A prediction operation is currently in progress, are you sure you want to leave?"}
                 />
-            </div>
+            </div >
```

Comment:
I think we dont need to have a space here

Reply:
sure

Reply:
done